### PR TITLE
Specify reports directory

### PR DESCRIPTION
**Pull request by bensteinberg**
This should allow the specification of a base directory for reports from merging accounts, something like
```
invoke merge-duplicative-accounts --reports-dir=/tmp
```

### CODE REVIEW

File: perma_web/tasks/dev.py [hunk 1]
```diff
@@ -667,28 +668,32 @@ def populate_folder_cached_path(ctx, batch_size=500):
...
 RETAINED_USERS_CSV = 'merge_reports/retained_users.csv'
 
-def initialize_csvs():
+
+def initialize_csvs(reports_dir):
+    p = Path(reports_dir)
+
     for filename in [TRANSFERRED_ORG_LINKS_CSV, TRANSFERRED_PERSONAL_LINKS_CSV, MERGED_USERS_CSV, RETAINED_USERS_CSV]:
-        os.makedirs(os.path.dirname(filename), exist_ok=True)
+        os.makedirs(p / os.path.dirname(filename), exist_ok=True)
```
Comment (rebeccacremona) [inline perma_web/tasks/dev.py]: I haven't used the `Path` stuff yet; love this syntax.

File: perma_web/tasks/dev.py [hunk 2]
```diff
@@ -667,28 +668,32 @@ def populate_folder_cached_path(ctx, batch_size=500):
...
 
+
 #
 # Merge user accounts
 #
 
-TRANSFERRED_ORG_LINKS_CSV = 'merge_reports/tranferred_org_links.csv'
-TRANSFERRED_PERSONAL_LINKS_CSV = 'merge_reports/tranferred_personal_links.csv'
+TRANSFERRED_ORG_LINKS_CSV = 'merge_reports/transferred_org_links.csv'
+TRANSFERRED_PERSONAL_LINKS_CSV = 'merge_reports/transferred_personal_links.csv'
```
Comment (rebeccacremona) [inline perma_web/tasks/dev.py:677]: Oh brother. I am the worst 🤣

**Review by rebeccacremona — APPROVED** [review]
Thanks! This is a much better plan!

File: perma_web/tasks/dev.py (hunk 2 above)
Comment (bensteinberg) [reply perma_web/tasks/dev.py:677]: (I almost forgot to fix these.)

File: perma_web/tasks/dev.py (hunk 1 above)
Comment (lizadaly) [reply perma_web/tasks/dev.py]: Once you start using `pathlib` you can largely forget about `os`:

```python
>>> from pathlib import Path
>>> p = Path('/tmp/reports/')
>>> filename = "merge_reports/merged_users.csv"
>>> report = p / filename 
>>> report.parent.mkdir(parents=True, exist_ok=True) # Creates /tmp/reports/merge_reports/
>>> report.write_text("hello world")  # Creates /tmp/reports/merge_reports/merged_users.csv as writable 
>>> report.read_text()
'hello world'
```

Comment (bensteinberg) [reply perma_web/tasks/dev.py]: Hm maybe I will switch to `parent.mkdir(parents=True, exist_ok=True)` here.

Comment (bensteinberg) [reply perma_web/tasks/dev.py]: (Done in a4bab73. I might switch to pathlib for the rest of this file, but that'll be a later PR.)

**Pull request merged**